app/libtest.sh

```diff
@@ -256,7 +258,20 @@ setupConode(){
 runCoBG(){
     for nb in "$@"; do
     	testOut "starting conode-server #$nb"
-    	( ./conode -d $DBG_SRV -c co$nb/private.toml | tee $COLOG$nb.log & )
+    	(
+	    rm -f "$COLOG$nb.log.dead"
+	    ./conode -d $DBG_SRV -c co$nb/private.toml server 2>&1 | tee "$COLOG$nb.log"
+	    touch "$COLOG$nb.log.dead"
```

Comment:
would this leave extra files after the test completes?

Reply:
the current working directory is the build dir, so it is cleaned up (or not, if -nt is given as an argument)